inductiva/tasks/output_consts.py

```diff
@@ -0,0 +1,43 @@
+"""Constants for connecting tasks to their simulation output classes."""
```

Comment:
Suggestion: Definition of constants and configurations for connecting tasks to their simulation output classes.